### PR TITLE
feat: add support vor mapbox vector tiles

### PR DESCRIPTION
**Pull request by hwbllmnn**
Adds support for Mapbox vector tile layers in the background chooser. In case a `LayerGroup` layer is found which has the flag `isVectorTile` set to `true`, the group layer's `url` value is used to create a vector tile group layer for the preview/overview.

@terrestris/devs Please review.

### CODE REVIEW

File: src/BackgroundLayerPreview/BackgroundLayerPreview.tsx [hunk 1]
```diff
@@ -110,8 +127,8 @@ export const BackgroundLayerPreview: React.FC<BackgroundLayerPreviewProps> = ({
       return;
     }
 
-    const newBgLayer = mainMap?.getLayerGroup().getLayers().getArray()
-      .find(l => getUid(l) === layerId);
+    const newBgLayer = MapUtil.getAllLayers(mainMap)
+      .find(l => getUid(l) === layerId) as any;
```
Comment (ahennr) [inline src/BackgroundLayerPreview/BackgroundLayerPreview.tsx:131]: Can typing be more specific here? eg. `OlLayer` or `OlLayerBase`?

**Review by ahennr — APPROVED** [review]

File: .github/workflows/on-pull-request.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ jobs:
           ${{ runner.OS }}-
 
     - name: Install dependencies ⏬
-      run: npm ci
+      run: npm ci --force
```
Comment (dnlkoch) [inline .github/workflows/on-pull-request.yml:32]: Why is this needed?

File: src/BackgroundLayerChooser/BackgroundLayerChooser.tsx [hunk 3]
```diff
@@ -88,6 +90,15 @@ export const BackgroundLayerChooser: React.FC<BackgroundLayerChooserProps> = ({
         ovLayer = new OlLayerImage({
           source: selectedLayer.getSource()
         });
+      } else if (selectedLayer instanceof LayerGroup) {
+        if (selectedLayer.get('isVectorTile')) {
```
Comment (dnlkoch) [inline src/BackgroundLayerChooser/BackgroundLayerChooser.tsx:94]: Where is `isVectorTile` coming from? Is this a custom attribute? If so, it needs to be documented.

File: src/BackgroundLayerPreview/BackgroundLayerPreview.tsx (hunk 1 above)
Comment (hwbllmnn) [reply src/BackgroundLayerPreview/BackgroundLayerPreview.tsx]: Right, for some reason I had that in there, but it's not necessary. Removed.

File: .github/workflows/on-pull-request.yml (hunk 2 above)
Comment (hwbllmnn) [reply .github/workflows/on-pull-request.yml:32]: `ol-mapbox-style` allows only for `ol` `<= 7.5.1`, but we have `7.5.2`...

File: src/BackgroundLayerChooser/BackgroundLayerChooser.tsx (hunk 3 above)
Comment (hwbllmnn) [reply src/BackgroundLayerChooser/BackgroundLayerChooser.tsx:94]: You're right, do you have a suggestion where to document this?

File: .github/workflows/on-pull-request.yml (hunk 2 above)
Comment (dnlkoch) [reply .github/workflows/on-pull-request.yml:32]: Alright, could you please add a comment to this line?

File: src/BackgroundLayerChooser/BackgroundLayerChooser.tsx (hunk 3 above)
Comment (dnlkoch) [reply src/BackgroundLayerChooser/BackgroundLayerChooser.tsx:94]: Probably it's best to add an appropriate tsdoc entry to the component (that should be catched up by the doc generator/the styleguide or whatever we might want to introduce later on).

Comment (hwbllmnn) [reply src/BackgroundLayerChooser/BackgroundLayerChooser.tsx:94]: Documented in https://github.com/terrestris/react-util/pull/478
